NOTICE-binary

```diff
@@ -5,411 +5,88 @@ Copyright 2022-2023 The Apache Software Foundation.
-   which is Apache license: http://www.apache.org/licenses/LICENSE-2.0
-
-   The PBE key processing routine PBECipher.createCipher() is adopted from http://juliusdavies.ca/svn/not-yet-commons-ssl/tags/commons-ssl-0.3.10/src/java/org/apache/commons/ssl/OpenSSL.java
-    which is also Apache APL-2.0 license: http://www.apache.org/licenses/LICENSE-2.0
-
-The binary distribution of this product bundles binaries of
-snakeyaml (https://bitbucket.org/asomov/snakeyaml),
-which has the following notices:
- * Copyright (c) 2008, http://www.snakeyaml.org
-
-The binary distribution of this product bundles binaries of
-metrics-core 3.2.4
-which has the following notices:
- * Copyright 2010-2013 Coda Hale and Yammer, Inc.
-
-   This product includes software developed by Coda Hale and Yammer, Inc.
-
-   This product includes code derived from the JSR-166 project (ThreadLocalRandom, Striped64,
-   LongAdder), which was released with the following comments:
-
-   Written by Doug Lea with assistance from members of JCP JSR-166
-   Expert Group and released to the public domain, as explained at
-   http://creativecommons.org/publicdomain/zero/1.0/
-
-htrace-core4
-Copyright 2016 The Apache Software Foundation
-
-This product includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
-
-Apache Log4j 1.x Compatibility API
-Copyright 1999-2022 The Apache Software Foundation
-
-This product includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
-
-Apache Log4j API
-Copyright 1999-2022 The Apache Software Foundation
-
-This product includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
-
-Apache Log4j Core
-Copyright 1999-2012 Apache Software Foundation
-
-This product includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
-
-Apache Log4j SLF4J Binding
-Copyright 1999-2022 The Apache Software Foundation
-
-Apache Ratis Client
-Copyright 2017-2020 The Apache Software Foundation
-
-Apache Ratis Common
-Copyright 2017-2020 The Apache Software Foundation
-
-Apache Ratis gRPC Support
-Copyright 2017-2020 The Apache Software Foundation
-
-Apache Ratis Metrics
-Copyright 2017-2020 The Apache Software Foundation
-
-Apache Ratis Netty Support
-Copyright 2017-2020 The Apache Software Foundation
-
-Apache Ratis Protocols
-Copyright 2017-2020 The Apache Software Foundation
-
-Apache Ratis Server
-Copyright 2017-2020 The Apache Software Foundation
-
-Apache Ratis Server API
-Copyright 2017-2020 The Apache Software Foundation
-
-Apache Ratis Thirdparty Miscellaneous
-Copyright 2017-2020 The Apache Software Foundation
-
----
-
-This product bundles protobuf-java which includes the following:
-
-Copyright 2008 Google Inc.
-
----
-
-This product bundles gRPC which includes the following text:
-
-Copyright 2014 The gRPC Authors
-
----
+------------------------------------------------------------------------------------
```

Comment:
Not sure if it's the proper way, but I think it's OK if we respect what Apache Flink does.

Reply:
9 to 92 lines need to be put into the License file, as they are license related.

Reply:
Updated. How about now?